cmd/gen-test-vectors/main.go

```diff
@@ -62,18 +63,30 @@ hnGbXDPbdFlL1nmayhnqyEfRdXNlpBT2U9hXcSxliKI1rHrAJFDx3ncttA==
 func main() {
 	flag.Parse()
 	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
-
 	env, err := integration.NewEnv(ctx)
 	if err != nil {
 		glog.Fatalf("Could not create Env: %v", err)
 	}
-	defer env.Close()
 	cctx, cancel2 := context.WithCancel(ctx)
 	if err := GenerateTestVectors(cctx, env); err != nil {
 		glog.Fatalf("GenerateTestVectors(): %v", err)
 	}
 	cancel2()
+	env.Close()
+	cancel()
+
+	ctx2, cancel := context.WithCancel(context.Background())
```

Comment:
Observation: 
This main function is beginning to look more like `impl/integration/test_main.go` 

We could make this a loop so we don't have to copy boiler plate code.